backend/inspirehep/search/facets.py

```diff
@@ -225,7 +225,7 @@ def citation_summary():
         "citation_count_without_self_citations",
     ]
     filters = get_filters_without_excluded(hep_filters(), excluded_filters)
-    if request.values.get("exclude-self-citations", False):
+    if json.loads(request.values.get("exclude-self-citations", "false")):
```

Comment:
what will happen if ``exclude-self-citations=+/1``? Can we use somehting like this https://github.com/inspirehep/inspirehep/blob/master/backend/inspirehep/search/views.py#L20 which we specify the type?

Reply:
By default all parameters in request.values are string already. 1/0 will work as True/False as it will be parsed by json.load as int. `+` would not work. It will return `500` as loading json will just raise

Reply:
Ah ok , I get what you meant. You are right. It's better this way. :+1:

Reply:
:-1: